.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Python package
+
+on: [ push, pull_request ]
+
+jobs:
+  unittests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+    name: Tests for Python ${{ matrix.python-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
+          activate-environment: formulate-env
+
+      - name: Install ROOT
+        run: |
+          conda config --add channels conda-forge
+          conda config --set channel_priority strict
+          conda install root -y
+
+      - name: Install test dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install coveralls
+          pip install pytest-cov
+          pip install -e .[dev,test]
```

Comment:
If you are using conda, you should probably install all the dependencies with conda, too, not pip.

Reply:
It shouldn't matter I would say (correct?) but I switched for `coveralls` and `pytest-cov`. However for the installation of the package (and deps) I would keep the `pip` in order to make sure that the installation works. Do you agree?

Or could there be problems when pip updates a package installed by conda?